lib/h3core.js

```diff
@@ -54,6 +54,13 @@ const SZ_COORDIJ = H3.sizeOfCoordIJ();
  * @typedef {string} H3Index
  */
 
+/**
+ * 64-bit hexidecimal string representation of an H3 index,
+ * or two 32-bit integers in little endian order in an array.
+ * @static
+ * @typedef {string | number[]} H3IndexInput
```

Comment:
Note that these get translated to Typescript defs, so we'll need to be sure they work as expected for TS (run `check-tsd`).

I think this would be better as `[number, number]` (a two-element tuple).

Reply:
I would also prefer `[number, number]` and actually added a check for the length, too. For some reason that type is not accepted by the TypeScript compiler, perhaps it needs to be expressed in a different way for the JSDoc style definitions?

Reply:
I suspect this is because JSDoc doesn't support it, not because TS doesn't. See https://github.com/jsdoc/jsdoc/issues/1703

It looks like there's an ugly workaround using `number[] & { 0: number, 1: number, length: 2 }`, but I don't see that this would be great for legibility, so I don't think it's worth pursuing here.